apps/climatemappedafrica/src/payload/fields/HURUMapURL/index.js

```diff
@@ -0,0 +1,109 @@
+import { Button } from "payload/components/elements";
+import {
+  Label,
+  TextInput,
+  useField,
+  useFormFields,
+} from "payload/components/forms";
+import { createElement, useState, useEffect } from "react";
+
+// TODO: @kelvinkipruto Handle i18n
+function HURUMapURL(props) {
+  const {
+    admin: { description },
+    name,
+    path,
+  } = props;
+  const { value, setValue } = useField({ path });
+  const [isValid, setIsValid] = useState(null);
+  const [loading, setLoading] = useState(false);
+  const [isButtonDisabled, setIsButtonDisabled] = useState(true);
+  // eslint-disable-next-line no-unused-vars
+  const HURUMapAPIURLValid = useFormFields(([_, dispatch]) => dispatch);
```

Comment:
```suggestion
  const urlValid = useFormFields(([_, dispatch]) => dispatch);
```
Why such ugly names?